src/Uno.UI/Generated/3.0.0.0/Windows.UI.Xaml.Controls/MediaPlayerPresenter.cs

```diff
@@ -5,9 +5,9 @@ namespace Windows.UI.Xaml.Controls
 	#if false || false || NET46 || __WASM__
 	[global::Uno.NotImplemented]
 	#endif
-	public  partial class MediaPlayerPresenter : global::Windows.UI.Xaml.FrameworkElement
+	public  partial class MediaPlayerPresenter : Border
```

Comment:
You can just omit the base class from a partial definition

Reply:
It's not an omission, it's a change: I need a container for the video surface, not only a FrameworkElement

Reply:
Sure, that's fine. I'm just pointing out you could've deleted FrameworkElement here and left the base class unspecified. (Assuming you inherit from Border in the authored code.)